runtime/shared_heap.c

```diff
@@ -211,7 +211,7 @@ static pool* pool_acquire(struct caml_heap_state* local) {
         caml_mem_round_up_mapping_size(Bsize_wsize(POOL_WSIZE) * new_pools);
       new_pools = mapping_size / Bsize_wsize(POOL_WSIZE);
 
-      void* mem = caml_mem_map(mapping_size, 0);
+      void* mem = caml_mem_map(mapping_size, 0, "major heap");
```

Comment:
Maybe this could say how many pools it is?
Also how about some kind of sequence number here, so we could see the temporal distribution in `smaps`?

Reply:
I put in a sequence number - it sounds useful for debugging compaction, at least.

The number of pools is redundant with the size, so I left that one out.